modules/manifold/src/constants/constants.js

```diff
@@ -1,35 +1,35 @@
-import {FEATURE_TYPE} from '@mlvis/mlvis-common/constants';
+import {absoluteError, logLoss} from '@mlvis/mlvis-common/utils';
 
 export const COLORS = {
   PINK: '#ff0099',
   BLUE: '#818c81',
-  // TODO add more if we need more than two segment groups
-};
-
-// urls for sample data
-export const SAMPLE_DATA_S3 = {
-  REGRESSION: ['/manifold/feature.csv', '/manifold/pred_reg.csv'],
-  BIN_CLASSIFICATION: [
-    '/manifold/feature.csv',
-    '/manifold/pred_bin_0.csv',
-    '/manifold/pred_bin_1.csv',
-  ],
 };
 
 // "performance": log-loss for classification models, squared-log-error for regression
 // "actual": raw prediction value
 export const METRIC = {
-  PERFORMANCE: 'performance',
+  REGRESSION: {
+    ABSOLUTE_ERROR: {
+      name: 'absolute error',
+      description:
+        'absolute value of difference between predicted value and ground truth',
```

Comment:
nit: missing articles, the absolute value of the difference between the predicted value and (the) ground truth.
same for line 23.